### ISSUE TITLE
Format single column for coverage range

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC-web-app#1648]
In order to buy us more space on the report data tables, we could combine the Coverage Start Date column with the Coverage End Date column.

Current:
![image](https://cloud.githubusercontent.com/assets/1696495/19651500/1b5319d4-99db-11e6-8363-c0ede3596335.png)

However, we need to do so in a way that keeps it easy to visually scan through end dates. 

For example:
![image](https://cloud.githubusercontent.com/assets/1696495/19651490/0e546f58-99db-11e6-9acf-c0e3e79746cc.png)

@jenniferthibault any ideas?

---

**Completion criteria:**
- [x] Identify pattern for showing coverage ranges in a single column, or decide that we shouldn't do that
- [ ] Implement decision

**Comment by noahmanger** [comment]
cc @emileighoutlaw

**Comment by emileighoutlaw** [comment]
Lots of thoughts on dates here: https://github.com/18F/fec-style/issues/553

TL;DR: I propose we display these as:

> **Coverage period**
> 11/08/2016 to 11/09/2016

**Comment by noahmanger** [comment]
This sounds good. I can take on implementing this. I think we won't know if it works in this context, though, until we see how it reads. This is kind of a unique situation where the _second_ date is the most important and needs to be scannable.

**Comment by jenniferthibault** [comment]
Nothing substantial to add, but +1 to a little validation would be nice if we can squeeze it in. Reminds me that most updates to the charts haven't been re-tested yet.

**Comment by emileighoutlaw** [comment]
I'm feeling slammed, so I'd definitely appreciate if you could take this to implementation, Noah.
